### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dKosstenko**
- [DEMO LINK](https://dkosstenko.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://dkosstenko.github.io/layout_search-bar-airbnb/report/html_report/)
I would appreciate any suggestions

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,50 @@
...
+    method="post"
+    class="search-form search-form_large"
+    data-qa="big"
+
+    >
+      <img
+      src="./images/Search.svg"
+      alt="Search icon"
+      class="icon-search icon-search_size-L"
+      >
```
Comment (lerastarynets) [inline src/index.html:25]:
```suggestion
      <img
        src="./images/Search.svg"
        alt="Search icon"
        class="icon-search icon-search_size-L"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -10,7 +10,50 @@
...
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+    action="#"
+    method="post"
+    class="search-form search-form_large"
+    data-qa="big"
+
+    >
```
Comment (lerastarynets) [inline src/index.html:20]:
```suggestion
      action="#"
      method="post"
      class="search-form search-form_large"
      data-qa="big"
    >
```

File: src/index.html [hunk 3]
```diff
@@ -10,7 +10,50 @@
...
+      >
+    </form>
+
+    <form
+    action="#"
+    method="post"
+    class="search-form search-form_small"
+    data-qa="small"
+
+    >
```
Comment (lerastarynets) [inline src/index.html:43]:
```suggestion
    <form
        action="#"
        method="post"
        class="search-form search-form_small"
        data-qa="small"
      >
```

File: src/index.html [hunk 4]
```diff
@@ -10,7 +10,50 @@
...
+      <img
+       src="./images/Search.svg"
+       alt="Search icon"
+       class="icon-search icon-search_size-S"
+       >
```
Comment (lerastarynets) [inline src/index.html:48]:
```suggestion
      <img
         src="./images/Search.svg"
         alt="Search icon"
         class="icon-search icon-search_size-S"
       >
```

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls mind html formatting)

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  margin: 20px 0;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.input[type="search"] {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
```
Comment (VitaliyBodnar) [inline src/style.css:40]: `font-family: "Avenir", sans-serif` is duplicated
you need to set font-family only one time

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+.input_size-S {
+  height: 42px;
+  font-size: 14px;
+  padding: 0 10px 0 33px;
+}
+
+.input[type="search"]:hover,
+.input[type="search"]:focus {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
```
Comment (VitaliyBodnar) [inline src/style.css:63]: color 'rgba(0, 0, 0, 0.25)' is duplicated, please move all duplicated styles to the variables

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,89 @@
+/* Fonts */
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
```
Comment (VitaliyBodnar) [inline src/style.css:6]: 400 is a default value for the font-weight, so it's redundant here

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**